### PR TITLE
[bookworm] Update loganalyzer ignore in test_vxlan_ecmp_switchover.py

### PR DESCRIPTION
**Pull request by congh-nvidia**
### Description of PR

Summary:
The information of an syslog error which should be ignored in the test test_vxlan_ecmp_switchover.py is changed in bookworm. Update the ignore list.

### Type of change

- [x] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [ ] 202311
- [x] 202405

#### What is the motivation for this PR?
Update loganalyzer ignore in test_vxlan_ecmp_switchover.py for bookworm.
#### How did you do it?
Add the error regexp to the ignore list in the test.
#### How did you verify/test it?
Run test on master image, passed.

### CODE REVIEW

**Comment by bingwang-ms** [comment]
Seems to be an empty string issue. Can it be fixed in image?

**Comment by congh-nvidia** [comment]
Hi @bingwang-ms,
The newly added error ".*basic_string: construction from null is not valid.*" is actually the same one as ".*_M_construct null not valid.*", which was added to the ignore list in PR(https://github.com/sonic-net/sonic-mgmt/pull/9653). 
And also I see that this error is ignored on KVM testbed in PR(https://github.com/sonic-net/sonic-mgmt/pull/14151), but we also see this on physical platforms.

**Comment by bingwang-ms** [comment]
Please fix the conflict.

**Comment by bingwang-ms** [comment]
Retriggered PR test.

**Comment by bingwang-ms** [comment]
@siqbal1986 Can you please help review?

File: tests/vxlan/test_vxlan_ecmp_switchover.py [hunk 1]
```diff
@@ -46,23 +46,17 @@ def fixture_encap_type(request):
...
+        loganalyzer[rand_one_dut_hostname].ignore_regex.extend(
+            [
+                ".*Unaccounted_ROUTE_ENTRY_TABLE_entries.*",
+                ".*missed_in_asic_db_routes.*",
+                ".*Look at reported mismatches above.*",
+                ".*Unaccounted_ROUTE_ENTRY_TABLE_entries.*",
+                ".*'vnetRouteCheck' status failed.*",
+                ".*Vnet Route Mismatch reported.*",
+                ".*_M_construct null not valid.*",
+                ".*basic_string: construction from null is not valid.*"
```
Comment (siqbal1986) [inline tests/vxlan/test_vxlan_ecmp_switchover.py:58]: if this error appears on an actual device, do we want to ignore it. it was previously ignored only for kvm tests.

**Comment by wsycqyz** [comment]
@congh-nvidia Can you help address the question above? It seems a general issue. I found it in Cisco platform.

File: tests/vxlan/test_vxlan_ecmp_switchover.py (hunk 1 above)
Comment (congh-nvidia) [reply tests/vxlan/test_vxlan_ecmp_switchover.py:58]: Hi @siqbal1986, I don't quite understand the question. This PR is just to ignore the error ".*basic_string: construction from null is not valid.*" for all platforms(kvm and physical). We observed this on physical testbeds too.
And actually the new error is the same one as ".*_M_construct null not valid.*" which was already in the ignore list, in bookworm the message has been updated.

**Comment by congh-nvidia** [comment]
> @congh-nvidia Can you help address the question above? It seems a general issue. I found it in Cisco platform.

@wsycqyz I replied above, please review.

**Comment by wsycqyz** [comment]
@siqbal1986 Request your review again. Thank you very much!

**Comment by wsycqyz** [comment]
Kindly request your review. We saw the issue on other platform.

**Review by wsycqyz — APPROVED** [review]

**Comment by wsycqyz** [comment]
@congh-nvidia I approved this PR since siqbal1986 has no comment.
Please request someone else to merget this PR.

**Review by bingwang-ms — APPROVED** [review]

**Comment by bingwang-ms** [comment]
@siqbal1986 PR merged since no reply for the comment. Please let me know if you have any concern.

**Comment by congh-nvidia** [comment]
Hi @bingwang-ms, this PR(#15132) is the manual cherry-pick to 202405 for conflict resolving, please help merge.
Thanks.

**Pull request merged**